plugins/flashrom-i2c/fu-i2c-device.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2020 The Chromium OS Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
```

Comment:
all plugin code has to be LGPLV2+

Reply:
We have done dual license on one plugin (dell-dock), so if it's needed to have BSD style, it should be BSD and LGPLv2+